client/js/content.js

```diff
@@ -78,15 +90,14 @@ const addRatingsToTile = async (movieTileElement) => {
 
     if (tileParent) {
         tileParent.appendChild(getRatingDivElement('IMDB', ratings.imdbRating));
-        tileParent.appendChild(
-            getRatingDivElement('Rotten Tomato', ratings.rottenTomato)
+        tileParent.appendChild(getRatingDivElement('Rotten Tomato', ratings.rottenTomato)
         );
     }
 };
 
 const induceDelay = (elem, callback) => {
     elem.onmouseover = () => {
-        setTimeout(() => callback(elem), 2000);
+        setTimeout(() => callback(elem), 500);
```

Comment:
500 is too less. Standard user mouse over times is around 1 to 3 seconds. 500 milliseconds will be too quick. We need the div element to show up before we can add ratings.

Reply:
If you hover on 2-3 tiles and land at the 4th tile, then you will see all 4 ratings at the same tile. This is because async calls are made within less than 2 seconds. I checked the div element was loading and there was no issue with that tile hover. I can show the demo of the issue in class tomorrow

Reply:
yeah thats the case I was trying to fix. I wrote some other code which handles it gracefully.

Actually if you have fixed it then great. If this still happens, then I have a possible fix. Also, I wanted to discuss with the team as well that may be we will repeat ratings at multiple places. So lets c